pretixprint/app/src/main/java/eu/pretix/pretixprint/ui/SunmiSettingsFragment.kt

```diff
@@ -0,0 +1,90 @@
+package eu.pretix.pretixprint.ui
+
+import android.os.Bundle
+import android.text.TextUtils
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ArrayAdapter
+import android.widget.AutoCompleteTextView
+import android.widget.Button
+import com.evolis.libevolis.androidsdk.model.ASDK_INTRAY
+import com.evolis.libevolis.androidsdk.model.ASDK_OUTTRAY
+import com.google.android.material.switchmaterial.SwitchMaterial
+import com.google.android.material.textfield.TextInputEditText
+import com.google.android.material.textfield.TextInputLayout
+import com.zebra.sdk.common.card.enumerations.CardDestination
+import com.zebra.sdk.common.card.enumerations.CardSource
+import eu.pretix.pretixprint.R
+import eu.pretix.pretixprint.byteprotocols.LinkOSCard
+import org.jetbrains.anko.support.v4.defaultSharedPreferences
+
+
+class SunmiSettingsFragment : SetupFragment() {
```

Comment:
Looks unused, doesn't compile. Can be thrown away?

Reply:
Correct